accounts/templates/accounts/register.html

```diff
@@ -1,13 +1,15 @@
+{% load static %}
+
 <!DOCTYPE html>
 <html>
 <head>
-<link rel="stylesheet" type="text/css" href="brewstyle.css"></head>
+    <link rel="stylesheet" type="text/css" href="{% static 'accounts/brewstyle.css' %}"></head>
 
 
 <div id="contact-form">
 	<div>
 		<h1>NYPL Brew Crew</h1> 
-		<p>Need more coffee with colleagues in your life? Join NYPL Brew Crew! Every two weeks, you�ll be matched with a new Brew Crew member and sent an invite�<em>automagically</em> based on your Google Calendar availability�to grab a cup of joe.</p>
+		<p>Need more coffee with colleagues in your life? Join NYPL Brew Crew! Every two weeks, you'll be matched with a new Brew Crew member and sent an invite&mdash;<em>automagically</em> based on your Google Calendar availability&mdash;to grab a cup of joe.</p>

```

Comment:
Some of the characters here were causing unicode errors.

Reply:
Oh, bummer! Sorry about that. Thx for fixing up!